src/Exporter.php

```diff
@@ -38,6 +38,8 @@
 
 final readonly class Exporter
 {
+    private const MAX_SHORTENED_ITEMS = 10;
```

Comment:
May I suggest that we make this a private property instead and that we introduce a constructor that accepts an integer (with a default value of 10 in order to not break BC)? Then we should be able to make this configurable from PHPUnit's end.

Reply:
absolutely. :)